tel-search-connector/processes/PhoneDirectory.mod

```diff
@@ -23,6 +23,7 @@ Py0 f0 outParamDecl '<java.util.List<String> matches> result;' #txt
 Py0 f0 outParamTable 'result.matches=in.matches;
 ' #txt
 Py0 f0 callSignature search(String) #txt
+Py0 f0 @CG|tags connector #txt
 Py0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
just saw that this process has yet no description... therefore we don't see any when picking connectors.
via right click -> inscribe process -> description you can provide one that will be shown in the connector activity selection dialog
![Selection_1008](https://user-images.githubusercontent.com/15085693/141748670-318f5557-900e-41af-b574-dcee479c2343.png)
![Selection_1007](https://user-images.githubusercontent.com/15085693/141748678-fd8e73c2-071d-41e3-be15-1ca89ab171af.png)